divviup/src/main/java/org/divviup/android/Client.java

```diff
@@ -70,11 +146,57 @@ public void sendMeasurement(M measurement) throws IOException {
                         leaderConfigList.bytes,
                         helperConfigList.bytes,
                         timestamp,
-                        (Boolean)measurement
+                        (Boolean) measurement
                 );
             } else {
                 throw new IllegalArgumentException("measurement for Prio3Count must be a Boolean");
             }
+        } else if (this.vdaf == VDAF.PRIO3SUM) {
```

Comment:
nit: consider using a `switch` statement instead. the first response here shows the syntax: https://stackoverflow.com/questions/6391777/switch-on-enum-in-java